README.md

```diff
@@ -16,3 +16,45 @@ You can find all the commands documented [here](https://docs.yugabyte.com/previe
 
 You can find the example workflows documented [here](https://docs.yugabyte.com/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-examples/)
 
+## Running unit tests locally:
+This project uses [GinkGo](https://onsi.github.io/ginkgo/) for testing.
```

Comment:
Ginkgo is not restrictly required for testing though, Go has its own testing framework. 
But Ginkgo can offer some additional functionalities, so it's a good to have (but not must)